language/tools/move-cli/src/sandbox/utils/mod.rs

```diff
@@ -98,6 +100,41 @@ fn print_struct_with_indent(value: &AnnotatedMoveStruct, indent: u64) {
     }
 }
 
+// Print a struct with a specified outer indent
+fn print_struct_diff_with_indent(
+    value1: &AnnotatedMoveStruct,
+    value2: &AnnotatedMoveStruct,
+    indent: u64,
+) {
+    let indent_str: String = (0..indent).map(|_| " ").collect::<String>();
+    let prev_str = format!("{}", value1);
+    let new_str = format!("{}", value2);
+
+    let Changeset { diffs, .. } = Changeset::new(&prev_str, &new_str, "\n");
+
+    let mut t = term::stdout().unwrap();
```

Comment:
It feels like using `term` for this is a bit heavyweight. We normally use something like the [colored](https://docs.rs/colored/2.0.0/colored/) crate to output colored text, but I'm fine with using this as well.